index.js

```diff
@@ -0,0 +1,87 @@
+module.exports = {
+  parser: "@typescript-eslint/parser",
+
+  extends: ["plugin:react/recommended", "prettier/@typescript-eslint", "plugin:prettier/recommended", "prettier/react"],
+
+  plugins: ["simple-import-sort", "jest", "@typescript-eslint", "react-hooks", "react", "prettier", "import"],
+
+  ignorePatterns: ["coverage", "**/dist", ".cache-loader", "node_modules", ".yalc"],
+
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: "module",
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+
+  settings: {
+    react: {
+      version: "detect",
+    },
+  },
+
+  env: {
```

Comment:
This should go in overrides, like override for test should come up with jest environment, etc.

Reply:
🤷‍♂ is there much harm? If put into overrides, then each of our projects will need to define the overrides. We can do that, but just not sure if it's worth the effort?

Reply:
Yes. It essentially prevents you from using globals you should not be using, i.e. accessing window globals inside of node.js code, etc.
It's a pretty nifty stuff